test/serializers/associations_test.rb

```diff
@@ -145,6 +145,26 @@ def test_associations_custom_keys
         assert expected_association_keys.include? :writer
         assert expected_association_keys.include? :site
       end
+
+      def test_belongs_to_doesnt_load_record
+        post = Post.new({ id: 997 })
+
+        class << post
+          def blog_id
+            996
+          end
+          def blog
+            fail "should use blog_id"
+          end
+        end
+
+        post_serializer = Class.new(ActiveModel::Serializer) do
+          belongs_to :blog, embed: :ids
```

Comment:
The `embed: :ids` here is useless though, as by JSONAPI standard, the default is to include just Resource Identifiers (i.e. an object `{ data: { id: 1, type: 'post' } }`), and also, the `embed` option does not seem to exist anymore (as searching the codebase for `embed` yields only results in tests and comments.